spec/features/requests/request_spec.rb

```diff
@@ -1362,18 +1356,13 @@
           expect(confirm_email.html_part.body.to_s).to have_content("Chekhov, Anton Pavlovich")
         end
 
-        it 'displays an ark link for a plum item' do
```

Comment:
I agree, ticketing this would be good since these are not in play any longer. I removed some other online related tests from this already but missed this one.